stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/info/group/GroupChatInfoViewModelBinding.kt

```diff
@@ -0,0 +1,58 @@
+package io.getstream.chat.ui.sample.feature.chat.info.group
+
+import androidx.lifecycle.LifecycleOwner
+import io.getstream.chat.android.client.ChatClient
+import io.getstream.chat.android.client.events.NotificationChannelMutesUpdatedEvent
+import io.getstream.chat.android.client.subscribeFor
+import io.getstream.chat.ui.sample.feature.chat.info.ChatInfoItem
+import io.getstream.chat.ui.sample.feature.chat.info.OptionType
+
+fun GroupChatInfoViewModel.bindView(view: GroupChatInfoFragment, lifecycleOwner: LifecycleOwner) {
```

Comment:
It's interesting that we bind the ViewModel to the Fragment here, and not a custom View. I guess this is because we're not planning to ship this in the library? In that case though, the Fragment could probably just contain this code directly, I think that's what we're doing elsewhere in similar situations.

Reply:
I wanted to reduce number of lines in Fragment class but agree there is no sense doing that